### PR TITLE
OCM-9896 | ci: Automated id:75233

### PR DESCRIPTION
**Pull request by jerichokeyne**
**What this PR does / why we need it**: Automates another test case: OCP-75233

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story (OCM-xxxx)*:
Fixes #[OCM-9896](https://issues.redhat.com//browse/OCM-9896)

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Build
- [x] CI
- [ ] Documentation
- [ ] Performance
- [ ] Refactor
- [ ] Style
- [ ] Unit tests
- [ ] Subsystem tests

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [x] Relevant issues have been referenced.

### CODE REVIEW

File: tests/e2e/trusted_ips_test.go
Comment (radtriste) [inline tests/e2e/trusted_ips_test.go:13]:
```suggestion
var _ = Describe("Trusted IPs", func() {
```

File: tests/e2e/trusted_ips_test.go
Comment (radtriste) [inline tests/e2e/trusted_ips_test.go:26]:
```suggestion
	It("can list trusted IPs - [id:75233]", ci.Day1, ci.Medium, func() {
```

Comment (radtriste) [reply tests/e2e/trusted_ips_test.go:26]: btw, that should be a day1-supplemental or day2

File: tests/e2e/trusted_ips_test.go
Comment (radtriste) [inline tests/e2e/trusted_ips_test.go:30]: Init is already done when calling `exec.NewTrustedIPsService`, no need to redo it

```suggestion
		err = trustedIPsService.Init()
		Expect(err).ToNot(HaveOccurred())
```

**Comment by jerichokeyne** [comment]
@radtriste Can you take another look when you get a chance?

**Review by radtriste — APPROVED** [review]
/lgtm

**Comment by jerichokeyne** [comment]
@radtriste can you /ok-to-test this PR?

**Comment by jerichokeyne** [comment]
@radtriste do you know why this can't be merged yet? Konflux is saying `Red Hat Konflux is waiting for approval`

**Pull request merged**
